src/index.js

```diff
@@ -75,21 +75,21 @@ export async function createWindowsInstaller(options) {
   let { appDirectory, outputDirectory, loadingGif } = options;
   outputDirectory = p`${outputDirectory || 'installer'}`;
 
-  await jetpack.copyAsync(
-    p`${__dirname}/../vendor/Update.exe`,
-    p`${appDirectory}/Update.exe`);
+  const vendorPath = path.join(__dirname, "..", "vendor");
+  await jetpack.copyAsync(path.join(vendorPath, "Update.exe"), path.join(appDirectory, "Update.exe"));
 
-  let defaultLoadingGif = p`${__dirname}/../resources/install-spinner.gif`;
+  let defaultLoadingGif = path.join(__dirname, "..", "resources", "install-spinner.gif");
```

Comment:
Same. `path.resolve("../resources/install-spinner.gif");` I believe it works on windows. at least in REPL